src/WingetCreateCLI/PromptHelper.cs

```diff
@@ -269,6 +309,119 @@ public static void PromptList<T>(string message, object model, string memberName
             }
         }
 
+        private static List<string> FilterRestrictedFields<T>(T model, PropertyInfo[] properties)
```

Comment:
This function is complex enough that it needs a documentation header

Reply:
Added documentation headers for these 3 functions

Reply:
Thanks!